src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,22 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int ELIGIBLE_AGE = 35;
+    private static final int INDEX_OF_AGE_FROM = 0;
+    private static final int INDEX_OF_AGE_TO = 1;
```

Comment:
I think it to be better if you rename it like :
```suggestion
    private static final int INDEX_OF_YEAR_FROM = 0;
    private static final int INDEX_OF_YEAR_TO = 1;
```